chapters/data-analysis.tex

```diff
@@ -1,292 +1,488 @@
+\textbf{Data visualization}\sidenote{\url{https://dimewiki.worldbank.org/wiki/Data\_visualization}} \index{data visualization}
+is increasingly popular, but a great deal lacks in quality.\cite{healy2018data,wilke2019fundamentals}
+We attribute some of this to the difficulty of writing code to create them.
+Making a visually compelling graph would already be hard enough if you didn't have to go through many rounds of googling to understand a command.
+The trickiest part of using plot commands is to get the data in the right format.
+This is why the \textbf{Stata Visual Library} includes example data sets to use 
+with each do-file.
+
+Whole books have been written on how to create good data visualizations,
+so we will not attempt to give you advice on it.
+Rather, here are a few resources we have found useful.
 The Tapestry conference focuses on ``storytelling with data''.\sidenote{
-\url{https://www.youtube.com/playlist?list=PLb0GkPPcZCVE9EAm9qhlg5eXMgLrrfMRq}}
+	\url{https://www.youtube.com/playlist?list=PLb0GkPPcZCVE9EAm9qhlg5eXMgLrrfMRq}}
 \textit{Fundamentals of Data Visualization} provides extensive details on practical application;\sidenote{
-\url{https://serialmentor.com/dataviz}}
+	\url{https://serialmentor.com/dataviz}}
 as does \textit{Data Visualization: A Practical Introduction}.\sidenote{
-\url{http://socvis.co}}
+	\url{http://socvis.co}}
+Graphics tools like Stata are highly customizable.
+There is a fair amount of learning curve associated with extremely-fine-grained adjustment,
+but it is well worth reviewing the graphics manual\sidenote{\url{https://www.stata.com/manuals/g.pdf}}
+For an easier way around it, Gray Kimbrough's \textit{Uncluttered Stata Graphs} code is an excellent default replacement for Stata graphics that is easy to install.
+\sidenote{\url{https://graykimbrough.github.io/uncluttered-stata-graphs/}}
+If you are a R user, the \textit{R Graphics Cookbook}\sidenote{\url{https://r-graphics.org/}} 
+is a great resource for the most popular visualization package \texttt{ggplot}\sidenote{\url{https://ggplot2.tidyverse.org/}}. 
+But there are a variety of other visualization packages, 
+such as \texttt{highcharter}\sidenote{\url{http://jkunst.com/highcharter/}}, 
+\texttt{r2d3}\sidenote{\url{https://rstudio.github.io/r2d3/}}, 
+\texttt{leaflet}\sidenote{\url{https://rstudio.github.io/leaflet/}}, 
+and \texttt{plotly}\sidenote{\url{https://plot.ly/r/}}, to name a few.
+We have no intention of creating an exhaustive list, and this one is certainly missing very good references.
+But at least it is a place to start.
+
+\section{Exporting analysis outputs}
+
+% Exploratory analysis
+It's ok to not export each and every table and graph created during exploratory analysis. 
+Instead, we suggest running them into markdown files using RMarkdown or the different dynamic document options available in Stata. 
```

Comment:
sidenote to RMarkdown or explain it as we haven't mentioned it yet in this chapter

Reply:
I'd just remove the sentence

Reply:
Removed in 97166a9. It seems I have a very particular exploratory analysis workflow in mind.